hs_core/views/utils.py

```diff
@@ -44,6 +44,20 @@
 ACTION_TO_AUTHORIZE = ActionToAuthorize(0, 1, 2, 3, 4, 5, 6, 7)
 
 
+def json_or_jsonp(r, i, code=200):
+    if not isinstance(i, basestring):
+        i = json.dumps(i)
+
+    if 'callback' in r.REQUEST:
+        return HttpResponse('{c}({i})'.format(c=r.REQUEST['callback'], i=i),
+                            content_type='text/javascript')
+    elif 'jsonp' in r.REQUEST:
+        return HttpResponse('{c}({i})'.format(c=r.REQUEST['jsonp'], i=i),
+                            content_type='text/javascript')
+    else:
+        return HttpResponse(i, content_type='application/json', status=code)
+
+
```

Comment:
Aside from the myriad of unused tag inclusions this looks to be the only function of any substance across our resources. 
With this migrated to hs_core it would appear that the old code is no longer needed.